### DISCUSSION TITLE
Use group class in a CRF

### DISCUSSION THREAD

**Discussion by catrepte** [reliatec-gmbh/LibreClinica#392] [Q&A]
I am looking for a way to copy an assigned group class to an item on a CRF in order control display of other items on that form.
PS: Is there any other part of LC where the group class has an effect or is even shown other then data exports)?

**Comment by toskrip** [comment]
If the group class is part of export, it should be also part of RESTfull URL for subject casebook. You have to try it out, but I think it should be possible to fetch it from withing eCRF constructing javascript making RESTfull URL call, extracting the subject group value and placing it in item that you want your control logic to depend on. It would be similar to this:

http://trialdatasolutions.com/tds/howto/itemsfromotherevents.jsp

but you would not fetch item data from another event, just the group class of the subject. Group class is otherwise not used anywhere else for any type of conditional logic in the system. It only serves for filtering in subject matrix as far as I know.
